Makefile

```diff
@@ -52,13 +62,17 @@ extra-clean: clean
 	go clean -i $(PKG_GOX)
 	go clean -i $(PKG_GOLANGCI_LINT)
 
+# fmt will fix the golang source style in place, and returns an error code if any source code are changed.
 fmt:
-	@gofmt -s -l -w $(SRC)
+	@gofmt -s -l -w $(GO_SRC)
+
+check-fmt:
```

Comment:
We may as well incorporate this in the `lint` target via `golangci-lint run $(LINT_DIRS) -E gofmt`

Reply:
Sure, but how do we control which options gofmt will be running with?

Reply:
From the docs: `gofmt: Gofmt checks whether code was gofmt-ed. By default this tool runs with -s option to check for code simplification [fast: true, auto-fix: true]`
But I'm also fine with keeping the direct usage of gofmt here, if it's clearer :+1: